src/pages/Detailed/header/DetailedHeader.js

```diff
@@ -0,0 +1,158 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+import cx from 'classnames'
+import {
+  createSkeletonElement,
+  createSkeletonProvider
+} from '@trainline/react-skeletor'
+import { compose } from 'recompose'
+import Label from '@santiment-network/ui/Label'
+import Button from '@santiment-network/ui/Button'
+import ProjectIcon from '../../../components/ProjectIcon'
+import PercentChanges from '../../../components/PercentChanges'
+import WatchlistsPopup from '../../../components/WatchlistPopup/WatchlistsPopup'
+import { formatNumber } from '../../../utils/formatting'
+import ChartSignalCreationDialog from '../../../ducks/SANCharts/ChartSignalCreationDialog'
+import addSignalSvg from './../../../assets/signals/buttons/addSignal.svg'
+import addWatchlistSvg from './../../../assets/watchlist/buttons/watchlist.svg'
+import styles from './DetailedHeader.module.scss'
+
+const DIV = createSkeletonElement('div', 'pending-header')
+
+const DetailedHeader = ({
+  project = {
+    ticker: '',
+    name: '',
+    description: '',
+    slug: '',
+    priceUsd: 0,
+    percentChange7d: 0,
+    percentChange24h: 0,
+    totalSupply: 0
+  },
+  loading,
+  isLoggedIn,
+  isDesktop
+}) => {
+  const {
+    id,
+    name,
+    description,
+    slug,
+    ticker,
+    priceUsd,
+    percentChange24h,
+    totalSupply,
+    percentChange7d
+  } = project
+
+  return (
+    <>
+      <div className={styles.breadcrambs}>
+        <Link to='/projects' className={styles.breadcrambLink}>
+          Assets
+        </Link>{' '}
+        <span className={styles.breadcrambLink}>{'>'}</span>{' '}
+        <Link className={styles.breadcrambCurrent} to={`/projects/${slug}`}>
+          {name}
+        </Link>
+      </div>
+      <div className={styles.wrapper}>
+        <div className={styles.left}>
+          <DIV className={styles.logo}>
+            <ProjectIcon name={name || ''} ticker={ticker} size={40} />
+            <div className={styles.name}>
+              <h1>{name}</h1>
+              <DIV className={styles.description}>{description}</DIV>
+            </div>
+          </DIV>
+
+          <div className={styles.price}>
+            <div className={styles.priceUsd}>
+              {priceUsd && formatNumber(priceUsd, { currency: 'USD' })}
```

Comment:
price in night mode still is dark. Could you also, pls, change this bug?
![image](https://user-images.githubusercontent.com/24521041/63329825-564ec380-c33b-11e9-9493-055360a30307.png)